pkg/database/database_util.go

```diff
@@ -343,7 +343,7 @@ func runMigrations(db *Database) error {
 	}()
 
 	// Run migrations.
-	if err := db.RunMigrations(ctx); err != nil {
+	if err := db.MigrateTo(ctx, "", false); err != nil {
```

Comment:
Aren't these equivalent? Why the change

Reply:
Yea, that's why I dropped `.RunMigrations` 😄

Reply:
We had two functions that did the same thing.